jenkins_common.bash

```diff
@@ -28,6 +28,6 @@ source "${TOOLBOX}/bash/kaldi.bash"
 # Refresh toolbox
 pip uninstall --quiet --yes paderbox
 pip show paderbox
-pip install --quiet --user -e ${TOOLBOX}
+pip install --quiet --user -e ${TOOLBOX}[all]
```

Comment:
In our environment this should be without the `all`. All dependencies should be installed in the global environment.